CHANGELOG.md

```diff
@@ -178,7 +178,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
-* None.  
+* Using `repo push` now pushes to the current repo branch (`HEAD`) instead of `master`  
```

Comment:
this entry is still off, needs to be up on `master` section, please rebase again and fix the entry.